src/utils.js

```diff
@@ -32,7 +32,7 @@ export const reformatValueTag = (value) => {
 
 export const removeEmptyKeys = (obj) => {
   Object.keys(obj).forEach(
-    (key) => (obj[key] === undefined || obj[key] === "") && delete obj[key]
+    (key) => (obj[key] == null || obj[key] === "") && delete obj[key]
```

Comment:
I would advice against using == as having to remember transformation equivalences is really something, better to have something explicit as    obj[key] === null || obj[key] === undefined

Reply:
Of course, we can explicit every thing but `== null` is a short cut I use a lot when `undefined` or `nul should have same effect`